pizzeria-backend/Services/AddOnService.cs

```diff
@@ -37,6 +38,28 @@ public async Task<AddOn> GetAddOn(int Id)
 
             return addOn;
         }
+
+        public async Task<List<AddOn>> GetAllAddOns()
+        {
+            List<AddOn> addOns = await _context.AddOns
+                .Include(a => a.Category)
+                .Select(a => new AddOn
```

Comment:
Why this select?

Reply:
It loads the Category relations to addOn so I used it so it doesn't send it(the frontend also doesn't need it atm)